src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,176 @@
+package duke.command;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+
+
+    public static void main(String[] args) throws IOException {
+
+        List listItem = new List();
+        File file = new File("C:\\Users\\muham\\OneDrive\\Desktop\\NUS MATERIAL\\Sem 5\\CS2113\\IP\\ip\\src\\main\\java\\duke.txt");
+
+
+        BufferedReader br = new BufferedReader(new FileReader(file));
+
+        String st;
+        String taskDesc = "";
+        String dateDesc = "";
+        while ((st = br.readLine()) != null) {
+            String[] commandList = st.split("\\|");
+            try {
+                for (int i = 0; i < commandList.length; i++) {
+                    if (i == 2) {
+                        taskDesc = commandList[i];
+                    } else if (i == 3) {
+                        dateDesc = commandList[i];
+                    }
+                }
+                boolean checked = false;
+                if (commandList.length > 1) {
+                    if (!(commandList[1].equals("1") || commandList[1].equals("0"))) {
+                        throw new DukeException("Error reading 1 or 0, skipping to next line");
+                    }
+                    checked = commandList[1].equals("1");
+                }
+                if (commandList[0].equals("T")) {
+                    ToDo t = new ToDo(taskDesc);
+                    if (checked)
+                        t.markInstructionAsDone();
+
+
+                    listItem.addTask(t);
+                } else if (commandList[0].equals("D")) {
+                    Deadline u = new Deadline(taskDesc, dateDesc);
```

Comment:
"u" seems too vague of a name for the variable - consider putting in the proper format